README.md

```diff
@@ -77,6 +80,17 @@ def sub_function(data, **kwargs):
       print(f'I am a task doing stuff with an event: {event}')
 ```
 
+#### Subscription `prefix`
```

Comment:
Shouldn't we be putting this sort of documentation in RTD?

Reply:
Do you mean here?

https://github.com/mercadona/rele/pull/47/files#diff-ec9803a8e3588dfed144c3b60f46f1f1R119

Reply:
No, I mean we can put it in the index.rst page for now.  Later, we might want to have a configuration.rst page.